test/e2e/specs/components/HelpSection.js

```diff
@@ -0,0 +1,105 @@
+module.exports = {
+    'Common license pop up redirects to https://creativecommons.org/licenses/': function (browser) {
+        const devServer = browser.globals.devServerURL
+
+        browser
```

Comment:
Please check if there's a way to cut down on test run time by doing a single `.waitForElementVisible`, then asserting that various things exist and that they redirect properly, as waiting 5000ms for each link is costly time-wise.

Reply:
Ok, I'll work on that right now

Reply:
I tried using the before statement, but since I redirect to another page only the first test runs correctly. I then changed to a beforeEach, and now the first test waits for about 7000ms to render the page, and then the following ones wait for 28-32ms, is that ok?
![image](https://user-images.githubusercontent.com/20938170/66799474-9b304a00-eee7-11e9-941f-c24dee0322a7.png)

Reply:
Looks to me that pretty much the same code is being executed for each test as before. No worries, though! I was just curious as to whether it could be optimized.